package.json

```diff
@@ -41,7 +41,7 @@
     "@vue/cli-plugin-router": "~4.5.0",
     "@vue/cli-plugin-typescript": "~4.5.0",
     "@vue/cli-plugin-unit-jest": "~4.5.0",
-    "@vue/cli-service": "~4.5.0",
+    "@vue/cli-service": "4.5.15",
```

Comment:
Why this is needed?

Reply:
> Why this is needed?
It updated automatically. Changed it back to version "~4.5.0"